folly/io/IOBuf.cpp

```diff
@@ -136,9 +136,9 @@ void IOBuf::SharedInfo::invokeAndDeleteEachObserver(
   if (observerListHead && cb) {
     // break the chain
     observerListHead->prev->next = nullptr;
-    auto* entry = observerListHead;
+    auto entry = observerListHead;
```

Comment:
This change doesn't seem quite as appropriate. In the other changes, the type is already named on the RHS so it doesn't need to be copied to the LHS. Here, the type is not named on the RHS. If an author wants to be a little bit clearer that the type here is a pointer, I'm not sure that needs to be changed.

Reply:
Right. It's more consistent with the other changes though.